tests/Doctrine/Tests/DBAL/Schema/SchemaTest.php

```diff
@@ -386,9 +386,6 @@ public function testVisitsVisitor(): void
             ->method('acceptTable')
             ->with($schema->getTable('bla.bloo'));
 
-        $visitor->expects($this->exactly(2))
-            ->method('acceptTable');
-
```

Comment:
Why is this change necessary?

Reply:
Because combined with `$this->at()` above it will cause the following warning on PHPUnit 9.3:

> More than one invocation handler has been configured for Doctrine\DBAL\Schema\Visitor\Visitor::acceptTable()

At the time of making the change, I thought that the combination of `$this->at()` calls will also define the expected count of calls but it doesn't limit the upper boundary. E.g. if I add one more call to the mock in the code being tested, with this condition remove, the test will pass but it should fail.

Let me see if there's a better way to fix this.

Reply:
Filed https://github.com/sebastianbergmann/phpunit/issues/4291.

Reply:
Very interesting outcome!

Reply:
Yeah, it's always fun to test with dev-stability dependencies.